README.md

```diff
@@ -24,6 +24,11 @@ $ npm i
 $ npm start
 ```
 
+## Running Tests
+
+1. `./node_modules/.bin/mocha test/**/*.js` to run all tests.
+2. `./node_modules/.bin/istanbul cover _mocha test/**/*.js` for test coverage report.
```

Comment:
I guess this should be a script as well. Maybe `coverage`? Not sure what's standard.